challenges/functions.js

```diff
@@ -6,20 +6,33 @@
   * The last parameter accepts a callback 
   * In the body of the function return the callback with the two parameters that you created
 */
+function consume(t, l, cb) {
+  return cb(t, l);
+}
 
 
 /* Step 2: Create several functions to callback with consume();
   * Create a function named add that returns the sum of two numbers
   * Create a function named multiply that returns the product of two numbers 
   * Create a function named greeting that accepts a first and last name and returns "Hello first-name last-name, nice to meet you!"
 */
-
+function add(t, l) {
+  return t + l;
+}
+function multiply(t, l) {
+  return t * l;
+}
+function greeting(t, l) {
+  return `Hello ${t} ${l}, nice to meet you!`;
+}
 
 /* Step 3: Check your work by un-commenting the following calls to consume(): */
 // consume(2,2,add); // 4
 // consume(10,16,multiply); // 160
 // consume("Mary","Poppins", greeting); // Hello Mary Poppins, nice to meet you!
-
+consume(2,2,add);  //4
+consume(10,16,multiply);  160
+consume("Mary","Poppins", greeting); Hello Mary Poppins, nice to meet you!
 
```

Comment:
this is the fix we talked about in your 1:1
console.log(consume(2,2,add));  //4
console.log(consume(10,16,multiply));  160
console.log(consume("Mary","Poppins", greeting));